data/fields/healthcare/speciality.json

```diff
@@ -1,5 +1,94 @@
 {
     "key": "healthcare:speciality",
     "type": "semiCombo",
-    "label": "Specialties"
+    "label": "Specialties",
+    "strings": {
+        "options": {
+            "abortion": "Abortion",
+            "acupuncture": "Acupuncture",
+            "allergology": "Allergy & Asthma",
```

Comment:
Some of the other entries spell out “and”, though the ampersand is used in some other fields’ strings. Whichever one is fine as long as it’s consistent.

Reply:
Ampersands are nice and short.